app/controllers/tasks_controller.rb

```diff
@@ -231,6 +226,11 @@ def load_and_authorize_task
     authorize @task
   end
 
+  def load_and_authorize_account_link
+    @account_link = AccountLink.find(params[:account_link])
+    authorize @account_link, :update?
```

Comment:
Why are you checking for `:update?` permissions here? It seems to be semantically incorrect and also prevents the use of shared account links.

Note: I think we cannot share an account link currently with any further users, but we still have the data model and existing account links present and can use them.

Would it make sense to add another verb for the usage, so that we do not overload `:show?`? Or shall we simply keep using it?